podman/domain/images.py

```diff
@@ -46,7 +52,7 @@ def history(self) -> List[Dict[str, Any]]:
         raise APIError(body["cause"], response=response, explanation=body["message"])
 
     def save(
-        self, chunk_size: Optional[int] = 2097152, named: Union[str, bool] = False
+        self, chunk_size: Optional[int] = 2 * 1024 * 1024, named: Union[str, bool] = False
```

Comment:
Thanks for spelling out the value-math, it really helps people understand where the value came from.